.github/labeler-issue.yml

```diff
@@ -0,0 +1,12 @@
+api:
+  - '(Devopness API)'
+website:
+  - '(Devopness website)'
+web-app:
+  - '(Devopness web app)'
+docs:
+  - '(Documentation)'
+sdk-js:
+  - '(SDK-JS)'
+other:
+  - "(Don't know / other)"
```

Comment:
Is there any real advantages of adding a label `other`?
Perhaps just ignore it, as the label `needs-triage` will be already added anyway?

Reply:
I see. `other` label removed.